viktorsha/Frontend_Tasks
# V1-01. Added a Datagrid responsive to row selection

You can check out the site by following [this link](https://viktorsha.github.io/Frontend_Tasks/).
### **Overview** ###
There's a Datagrid with static data on the site. It contains 4 columns: _id_ of the note, _Title_, _Description_, and _Creation date_. 
One can select the row with specific data and check it out fully on the card placed on the right side of the page. 
The card contains full info about the note.

### **Changes** ###
Created a React application from scratch.

### **How to test** ###
1. Open the page by following [this link](https://viktorsha.github.io/Frontend_Tasks/)
2. Once the page is loaded a card on the right side should contain the text "Select note to display"
3. As soon as one selects the row, information from it should appear on the card.
